<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="butPos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setPos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonComOpen">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPos">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setVel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetRange">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setRange</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setDir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDir">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRange3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetCont">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setCont</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetCont_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setCont</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetRange_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setRange</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVel_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRange3_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setVel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPos_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDir_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setDir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDir_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="butPos_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>setPos</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>22</width>
      <height>181</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>22</width>
      <height>191</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3999</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>891</width>
      <height>461</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextUDP">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>220</y>
      <width>191</width>
      <height>461</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="verticalMotorPos1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalMotorPos2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalMotorPos3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalMotorPos4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalMotorPos5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalMotorPos6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalMotorPos7">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalMotorPos8">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalMotorPos9">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalMotorPos10">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refreshCom">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
